### DISCUSSION TITLE
Publii 0.45 Windows - Images in preview are not displayed

### DISCUSSION THREAD

**Discussion by okara90** [GetPublii/Publii#1416] [Help] [answered]
In version 0.45, images are no longer displayed in the preview, the "media" folder is not generated. What am I doing wrong?

**Comment by bobmitro** [comment]
More details? What OS? When did you download the files?

**Reply by okara90** [reply]
Windows 11. In general, the problem is observed with both the old images and the ones I uploaded in the test post. 


I noticed that changes were made to the preview mode - https://github.com/GetPublii/Publii/issues/1306. It is possible that this problem is related to those changes.


If I understand everything correctly, the images should now be loaded from the input folder (or not?), but this is not happening for some reason.

**Reply by bobmitro** [reply]
When did you download the files? morning?

**Reply by okara90** [reply]
If you are referring to the Publii app, I downloaded it an hour ago.

**Comment by giampieropace** [comment]
Same here, WIN 10, downloaded 2hrs ago.

In preview the media file is not created, but if you publish (e.g.: Manual upload set up), the media folder is regularly created.

**Comment by bobmitro** [comment]
It looks like a permissions issue; the media folder is removed from the preview, but Windows blocks the creation of a symlink. 
We need to generate a new package for Windows without this functionality.

**Comment by bobmitro** [comment, marked as answer]
Please redownload the Windows version, build 16607 (refresh the cache of the download page: https://getpublii.com/download/).

**Comment by giampieropace** [comment]
Now it works again, thanks Bob!

**Comment by bobmitro** [comment]
FYI: We have just released a hotfix v0.45.1. However, if you are using the Windows version v0.45.0 build 16607, you do not need to install it.
